packages/babel-helper-module-transforms/src/index.js

```diff
@@ -150,6 +155,30 @@ export function buildNamespaceInitStatements(
   return statements;
 }
 
+export function buildReexportsFromMeta(meta, metadata, loose) {
+  const reexports = [];
+  for (const [exportName, importName] of metadata.reexports) {
```

Comment:
wouldnt a `.map` be better describing the intent here?

Reply:
@Andarist Sure! Thanks

Reply:
@Andarist I've checked.. metadata.reexports is a Map. I think, it's better to use `for..of` instead of `Array.from` because of performance and consistency.

Reply:
Not sure... a `return Array.from(metadata.reexports, ([exportName, importName]) => (loose ? ... : ...))` looks better to me, but I just overall prefer functional style 🤔